modules/virt-about-runstrategies-vms.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-runstrategies-vms_{context}"]
+= About RunStrategies for virtual machines
+
+A `RunStrategy` for virtual machines determines a virtual machine instance (VMI)'s behavior, depending on a series of conditions. The `spec.runStrategy` setting exists in the virtual machine configuration process as an alternative to the `spec.running` setting.
```

Comment:
I think that the `'s` should be on the written out "virtual machine instance", not the acronym.